python/cudf/cudf/core/column/numerical.py

```diff
@@ -448,6 +449,8 @@ def find_first_value(self, value, closest=False):
         columns, returns the offset of the first larger value
         if closest=True.
         """
+        if not pd.api.types.is_number(value):
```

Comment:
May be we have to handle 0-d array as well.

```
>>> import pandas as pd
>>> import numpy as np
>>> pd.Series([1, 2, 3, 4, 5])[np.array(3)]
4

```

Reply:
`np.array([3])` is a 1-dim array. `np.array([3])[0]` is a 0-dim array. It looks like both just work right now:

```python
In [4]: a = cudf.Series([1, 2, 3])

In [5]: a[np.array([0])]
Out[5]:
0    1
dtype: int64

In [6]: a[np.array([0])[0]]
Out[6]: 1
```

Reply:
@shwina try with `np.array(0)`

Reply:
Sorry! I misread. I added handling 0-dim arrays and a test!